### PR TITLE
README: More vfkit updates

### PR DESCRIPTION
**Pull request by cfergeau**

### CODE REVIEW

**Comment by baude** [comment]
lgtm

**Comment by cfergeau** [comment]
(based on Brent's lgtm, this is only a README change - I've fixed the missing DCO reported by CI)
/lgtm
/approve

**Review by gbraad — APPROVED** [review]
/appove

File: README.md [hunk 1]
```diff
@@ -51,9 +51,18 @@ Starting from Qemu version 7.2.0 it is possible to run w/o a wrapper:
...
 More docs about the User Mode Linux with BESS socket transport: https://www.kernel.org/doc/html/latest/virt/uml/user_mode_linux_howto_v2.html#bess-socket-transport
 
+## Run with [vfkit](https://github.com/crc-org/vfkit)
+
+With vfkit 0.1.0 or newer, gvproxy can be used without any helper running in the VM:
+
+```
+(terminal 1) $ bin/gvproxy -debug -listen unix:///tmp/network.sock --listen-vfkit unixgram:///tmp/vfkit.sock
+(terminal 2) $ vfkit (all your vfkit options) --device virtio-net,unixSocketPath=/tmp/vfkit.sock,mac=5a:94:ef:e4:0c:ee
+```
```
Comment (praveenkumar) [inline README.md:61]: 👍🏼

**Review by praveenkumar — APPROVED** [review]

**Pull request merged**